src/routes/Authentication.js

```diff
@@ -1,10 +1,21 @@
 import express from 'express';
 import { createAgent, loginAgent } from '../controllers/AgentAuthentication';
-import { validateUserInput } from '../middleware';
+import {
+  agentProperties, allProperties, createProperty,
+  deleteProperty, editProperty, getPropertyById
+} from '../controllers/propertyInfo/propertyController';
+import { validatePropertyInput, validateUserInput } from '../middleware/middleware';
+import { verifyToken } from '../middleware/auth';
 
 const indexRouter = express.Router();
 
 indexRouter.post('/signup', validateUserInput, createAgent);
 indexRouter.post('/login', loginAgent);
+indexRouter.get('/properties', allProperties);
+indexRouter.get('/property/:id', getPropertyById);
+indexRouter.post('/post-property', verifyToken, validatePropertyInput, createProperty);
+indexRouter.get('/post-property/:id', verifyToken, agentProperties);
```

Comment:
This is a get route, why named it `post-property` ? Since this is specific to agents, its better named `/agent/property`